CMSgov/easi-app
# [NOREF] Replace cedarCoreEnabled LD flag with an env var that enables mocking

## Changes and Description

- Replaces `cedarCoreEnabled` LaunchDarkly flag with `CEDAR_CORE_MOCK` env var.
- Replaces `emit-to-cedar` LaunchDarkly flag with `CEDAR_INTAKE_ENABLED` env var.

~~The choice to make these `true` to disable each API was made because the edge case (wanting to disable them locally) is less common / expected than the case of wanting to actually hit the API (each deployed environment). Definitely happy to re-visit this decision if we think it'd make more sense the other way around -- we just have to remember to update the S3 env vars if we go that route, so as not to disable CEDAR API calls in our deployed environments.~~ This is no longer the case, as these variables have been renamed since making this PR!

## How to test this change

Add the following to your .envrc.local, and run `direnv allow`
```bash
export CEDAR_CORE_MOCK=true
export CEDAR_INTAKE_ENABLED=false
```

When running the app locally, ensure that the `Systems` page populates with mock data instead of real data, and that submitting a new IT Gov (Intake) request doesn't result in a call to the CEDAR API (if you're not on the VPN, this would normally cause the submit request to hang!)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
